ci/pipeline.yml

```diff
@@ -75,9 +65,7 @@ jobs:
 - #@ testflight_job(chart, "bitcoin")
```

Comment:
In the PR https://github.com/GaloyMoney/galoy-charts/pull/4 , we removed `bitcoin_mattermost_webhook_url` which is one of the dependencies for `bitcoin` testflight. With the current setup we get an error:

```
invalid jobs:
	jobs.bitcoind-testflight.plan.do[6].put(bitcoin-mattermost): unknown resource 'bitcoin-mattermost'
	jobs.bitcoind-testflight.plan.on_failure.put(bitcoin-mattermost): unknown resource 'bitcoin-mattermost'
	jobs.bria-testflight.plan.do[6].put(bitcoin-mattermost): unknown resource 'bitcoin-mattermost'
	jobs.bria-testflight.plan.on_failure.put(bitcoin-mattermost): unknown resource 'bitcoin-mattermost'
	jobs.fulcrum-testflight.plan.do[6].put(bitcoin-mattermost): unknown resource 'bitcoin-mattermost'
	jobs.fulcrum-testflight.plan.on_failure.put(bitcoin-mattermost): unknown resource 'bitcoin-mattermost'
```

Do we re-introduce `bitcoin-mattermost` in `values.yml` or remove this line and fix subsequent error?

@krtk6160

Reply:
Tried introducing `bitcoin_mattermost_webhook_url` and the code related to it, fixes the repipe script and a paused pipeline can be found here : https://ci.galoy.io/teams/dev/pipelines/helm-charts